### PR TITLE
Migrate package admin page

### PR DESCRIPTION
**Pull request by isoos**
- first commit fixes the missing content from the golden file
- allows `viewBox` (cased) attribute on `svg` tag
- changes `<i>` to `<em>` in blocks that are now implemented with markdown
- we have the usual attribute order changes

### CODE REVIEW

File: app/lib/frontend/dom/dom.dart [hunk 1]
```diff
@@ -399,6 +399,32 @@ Node li({
       children: _children(children, child, text),
     );
 
+/// Creates an `<option>` Element using the default [DomContext].
+Node option({
+  String? id,
+  Iterable<String>? classes,
+  Map<String, String>? attributes,
+  Iterable<Node>? children,
+  Node? child,
```
Comment (jonasfj) [inline app/lib/frontend/dom/dom.dart:408]: Does it ever make sense to have children or child, isn't `text` the only thing worth using?

File: app/lib/frontend/templates/views/pkg/admin_page.dart [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+          children: [
+            d.p(
+                text: 'You can invite new uploader to this package. '
+                    'Once new uploaders accept the invitation, they have full administrative rights, '
+                    'with the following abilities:'),
+            d.ul(children: [
+              d.li(text: 'Transfer this package to a publisher'),
+              d.li(text: 'Upload new versions of this package'),
+              d.li(text: 'Invite and remove uploaders of this package'),
+            ]),
```
Comment (jonasfj) [inline app/lib/frontend/templates/views/pkg/admin_page.dart:99]: nit. Consider using markdown..

File: app/lib/frontend/dom/material.dart [hunk 3]
```diff
@@ -246,3 +246,70 @@ d.Node dataTable<T>({
...
+          ),
+          d.div(classes: ['mdc-checkbox__ripple']),
+        ],
+      ),
+      d.label(attributes: {'for': id}, text: label),
+    ],
+  );
+}
+
+/// Renders a material dropdown / select component.
```
Comment (jonasfj) [inline app/lib/frontend/dom/material.dart:295]: Let's document that `options` should be a list of `d.option`

**Review by jonasfj — APPROVED** [review]

File: app/lib/frontend/dom/dom.dart (hunk 1 above)
Comment (isoos) [reply app/lib/frontend/dom/dom.dart:408]: I think material and other frameworks allow the creation of complex selects, with icons or formatted content. Not sure if they need rich content or data attributes for that, but let's keep that open.

File: app/lib/frontend/templates/views/pkg/admin_page.dart (hunk 2 above)
Comment (isoos) [reply app/lib/frontend/templates/views/pkg/admin_page.dart:99]: Let's refactor that next time this part needs an update.

**Review by jonasfj — APPROVED** [review]

**Pull request merged**